### PR TITLE
example: added some examples

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description
Was added some examples,  i used the esp32 wrover-kit  board for make this.

1) ## esp-wrover-app

This example is a little app where  i showing the price of bitcoin and  ethereum in the screen of the esp32 i used a server and client udp and extrrnal server for get criptocurrencies value.

2) ## external_led
basic example where  I turn on and turn off some leds  connected in a protoboard

3) ## TPM_UART_APP
basic example where i used  hc-06 bluetooth and  temperature sensor.

### CODE REVIEW

**Comment by luisan00** [comment]
Good to see you @xkevin190 !!, some notes:

1.-  For the **PR** can you  please add a more detailed description in each case,.

  - Overall
  -  Platform supported and peripherals and other elements required for to replicate the example.
  - How to test it?

2.-  About your contribution:
  - A `README.md` for each example, describing the example, how to replicate it, peripherals or/and other external elements required... etc.
  - headers files need documentation
  - A doc.txt also can be useful for documentation purposes

File: examples/TPM_UART_APP/Makefile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+FEATURES_REQUIRED = periph_adc
+USEMODULE += ztimer
+USEMODULE += ztimer_msec
+USEMODULE += printf_float
+
+
+
+# If We need custom actions, routines, etc **SHOULD BE** included in the "core" folder
+# 	and "MUST BE" code hardware independent
+# EXTERNAL_MODULE_DIRS += $(CURDIR)/../core
```
Comment (luisan00) [inline examples/TPM_UART_APP/Makefile:29]: Delete dead lines of code

File: examples/TPM_UART_APP/Makefile [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+DEVHELP ?= 1
+QUIET ?= 1
+
+
```
Comment (luisan00) [inline examples/TPM_UART_APP/Makefile:34]: A blank line should be sufficient here

File: examples/TPM_UART_APP/Makefile [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+# Custom modules
+# Note: this code is hardware independent but "COULD BE" configurable for a
+#	target board
+# EXTERNAL_MODULE_DIRS += external_drivers
```
Comment (luisan00) [inline examples/TPM_UART_APP/Makefile:39]: Dead code ...

File: examples/TPM_UART_APP/Makefile [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
```
Comment (luisan00) [inline examples/TPM_UART_APP/Makefile:40]: Delete extra line

File: examples/TPM_UART_APP/Makefile [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+CFLAGS += -DCONFIG_GNRC_IPV6_NIB_NUMOF=50
+CFLAGS += -DCONFIG_GNRC_IPV6_NIB_OFFL_NUMOF=50
+#
+CFLAGS += -DCONFIG_GNRC_IPV6_NIB_DEFAULT_ROUTER_NUMOF=2
+
+# Disable warnings when a parameter is not used
+CFLAGS += -Wno-unused-parameter
+CFLAGS += -Wno-unused-variable
+
+
```
Comment (luisan00) [inline examples/TPM_UART_APP/Makefile:53]: Empty line

**Review by luisan00 — COMMENTED** [review]
It's a good job but  before to merge, please double check the code formatting, empty lines, dead code, etc

**Comment by xkevin190** [comment]
were removed the empty lines and were added readmes.

**Comment by luisan00** [comment]
Looks good :+1:

**Pull request merged**